src/v/cluster/self_test_rpc_types.cc

```diff
@@ -35,6 +35,24 @@ std::ostream& operator<<(std::ostream& o, self_test_status sts) {
     return o;
 }
 
+ss::sstring self_test_stage_as_string(self_test_stage sts) {
+    switch (sts) {
+    case self_test_stage::idle:
+        return "idle";
```

Comment:
nit: i'm not sure, but "idle" makes it sound like the test is running but not doing anything? i wonder if "not running" or something like that would be better? i don't think it's a big deal either way.